config/cert-manager/kustomization.yaml

```diff
@@ -21,4 +21,4 @@ helmCharts:
       cainjector:
         enabled: false
       # Customize active controllers
-      extraArgs: ["--controllers=*,-ingress-shim"]
+      extraArgs: ["--controllers=*,-ingress-shim",--enable-certificate-owner-ref=true]
```

Comment:
this ensure secrets are cleaned up when certificates are removed so we no longer need to do that in our controller